libs/platform/src/lib/components/dynamic-page/dynamic-page.component.ts

```diff
@@ -214,6 +217,19 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
         }
     }
 
+    /**
+     * marks the dynamic page tab as selected when the id of the tab is passed
+     */
+    setSelectedTab(id: string): void {
+        if (id && this.dynamicPageTabs) {
+            this.dynamicPageTabs.forEach((element) => {
+                if (element.id === id) {
+                    element.open(true);
+                }
+            });
+        }
```

Comment:
please decrease nesting
```suggestion
        if (!(id && this.dynamicPageTabs)) {
            return;
        }
        
        this.dynamicPageTabs.forEach((element) => {
            if (element.id === id) {
                element.open(true);
            }
        });
```

Reply:
fixed. Please check again.